src/main/steps/applicant1/review-the-application/content.ts

```diff
@@ -180,15 +192,24 @@ const languages = {
 };
 
 export const generateContent: TranslationFn = content => {
-  const translations = languages[content.language](content);
-  const isApplicantAddressNotPrivate = content.userCase.applicant1AddressPrivate !== YesOrNo.YES;
-  const isRespondentAddressNotPrivate = content.userCase.applicant2AddressPrivate !== YesOrNo.YES;
-  const isFinancialOrderYes = content.userCase.applicant1ApplyForFinancialOrder === YesOrNo.YES;
+  const { language, userCase } = content;
+  const translations = languages[language](content);
+  const isApplicantAddressNotPrivate = userCase.applicant1AddressPrivate !== YesOrNo.YES;
+  const isRespondentAddressNotPrivate = userCase.applicant2AddressPrivate !== YesOrNo.YES;
```

Comment:
why use negative, it's confusing

Reply:
tbh, that wasnt my code change, i just changed the variable for userCase but ill make it easier to read